### ISSUE TITLE
system error "Telemetry.dll missing"

### ISSUE THREAD

**Issue by jendrikseipp** [jendrikseipp/rednotebook#291]
Installed Rednotebook 1.10.1 -- when opening the program I keep getting system error "Telemetry.dll missing". After 3 notifications the error notice goes away and rednotebook is useable. I've tried reinstalling it, and I tried installing v. 1.10 but nothing resolves missing file. I'm running Windows 7, 64 bit.

**Comment by jendrikseipp** [comment]
*Original comment by* **jendrikseipp** (2015-06-16T11:24:01Z):

----------------------------------------

I don't think that RedNotebook depends on Telemetry.dll in any way. Your 
system may be misconfigured in some way.

**Comment by jendrikseipp** [comment]
*Original comment by* **gregpenn** (2015-06-16T12:20:51Z):

----------------------------------------


    
Thanks. That helps.


Sent from my Sprint Samsung Galaxy® Note 4.

-------- Original message --------
From: Jendrik Seipp <email address hidden> 
Date: 06/16/2015  7:24 AM  (GMT-05:00) 
To: <email address hidden> 
Subject: Re: [Bug 1465368] [NEW] system error "Telemetry.dll missing" 

I don't think that RedNotebook depends on Telemetry.dll in any way. Your 
system may be misconfigured in some way.

-- 
You received this bug notification because you are subscribed to the bug
report.
https://bugs.launchpad.net/bugs/1465368

Title:
  system error "Telemetry.dll missing"

Status in RedNotebook - A Desktop Journal:
  New

Bug description:
  Installed Rednotebook 1.10.1 -- when opening the program I keep
  getting system error "Telemetry.dll missing". After 3 notifications
  the error notice goes away and rednotebook is useable. I've tried
  reinstalling it, and I tried installing v. 1.10 but nothing resolves
  missing file. I'm running Windows 7, 64 bit.

To manage notifications about this bug go to:
https://bugs.launchpad.net/rednotebook/+bug/1465368/+subscriptions
